components/concurrency_manager/Cargo.toml

```diff
@@ -19,9 +19,10 @@ tokio = { version = "0.2", features = ["macros", "sync", "time"] }
 txn_types = { path = "../txn_types", default-features = false }
 fail = "0.4"
 
+# FIXME: switch to the crates.io version after crossbeam-skiplist is released
 [dependencies.crossbeam-skiplist]
-git = "https://github.com/crossbeam-rs/crossbeam.git"
-tag = "crossbeam-0.7.3"
+git = "https://github.com/sticnarf/crossbeam.git"
```

Comment:
Although the fix is already merged to the upstream, I use my own branch to make it possible to use the crates.io version of `crossbeam-utils` and `crossbeam-epoch`.

Reply:
just curious, what are the advantages of the crates.io version?

Reply:
I just don't want another copy of these dependencies...